src/zzz_od/gui/view/home/home_interface.py

```
@@ -637,23 +753,33 @@ def reload_banner(self, show_notification: bool = False) -> None:
-        elif self.ctx.custom_config.version_poster and os.path.exists(version_poster_path):
-            banner_path = version_poster_path
-        elif self.ctx.custom_config.remote_banner and os.path.exists(remote_banner_path):
-            banner_path = remote_banner_path
+        custom_banner_path = Path(os_utils.get_path_under_work_dir('custom', 'assets', 'ui')) / 'banner'
+        ui_dir = Path(os_utils.get_path_under_work_dir('assets', 'ui'))
+        official_dynamic_path = ui_dir / 'official_dynamic.webm'
+        version_poster_path = ui_dir / 'version_poster.webp'
+        remote_banner_path = ui_dir / 'remote_banner.webp'
+        index_banner_path = ui_dir / 'index.png'
+
+        # 主页背景优先级：自定义 > 枚举选项 > index.png
```

Ruff (0.14.1)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)